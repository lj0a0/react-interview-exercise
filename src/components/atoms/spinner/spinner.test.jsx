import React from "react";
import Spinner from "./spinner";
import renderer from "react-test-renderer";


test("Spinner snapshot visible", () => {
  const component = renderer.create(
    <Spinner
    loading={true}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Spinner snapshot hidded", () => {
    const component = renderer.create(
      <Spinner
      loading={false}
      />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });