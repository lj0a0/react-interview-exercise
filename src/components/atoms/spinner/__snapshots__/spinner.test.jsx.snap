// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner snapshot hidded 1`] = `
<div
  className="loading-container"
  id="loading"
  style={
    Object {
      "display": "none",
    }
  }
>
  <div
    className="d-flex justify-content-center align-items-center loader"
  >
    <div
      className="spinner-border text-primary"
      role="status"
    >
      <span
        className="sr-only"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;

exports[`Spinner snapshot visible 1`] = `
<div
  className="loading-container"
  id="loading"
  style={null}
>
  <div
    className="d-flex justify-content-center align-items-center loader"
  >
    <div
      className="spinner-border text-primary"
      role="status"
    >
      <span
        className="sr-only"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
